apps/pie-microsite/src/assets/styles/_utilities.scss

```diff
@@ -0,0 +1,5 @@
+@use 'fozzie-config' as f;
+
+.u-font-bold {
```

Comment:
Nit: should this be `u-font--bold`?

Reply:
Should only be `u-font--bold` if `u-font` existed as a classname in the project. Otherwise, it wouldn't be modifying a base class.

There's likely a case for it being called `u-fontBold` though instead

Reply:
I don't have a personal preference. which would you both prefer? :)

Reply:
I'm happy to leave this one as-is then